sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/RxGatewayStoreModelTest.java

```diff
@@ -82,14 +83,18 @@ public void readTimeout() throws Exception {
         Mockito.doReturn(Mono.error(ReadTimeoutException.INSTANCE))
                 .when(httpClient).send(any(HttpRequest.class), any(Duration.class));
 
+        GatewayServiceConfigurationReader gatewayServiceConfigurationReader = Mockito.mock(GatewayServiceConfigurationReader.class);
+        Mockito.doReturn(ConsistencyLevel.SESSION)
+            .when(gatewayServiceConfigurationReader).getDefaultConsistencyLevel();
         RxGatewayStoreModel storeModel = new RxGatewayStoreModel(clientContext,
                 sessionContainer,
                 ConsistencyLevel.SESSION,
                 queryCompatibilityMode,
                 userAgentContainer,
                 globalEndpointManager,
                 httpClient,
-                null);
+            null);
```

Comment:
nit: space seems a little bit off

Reply:
I didn't find space issue , may be git hub ui thing